tools/local-snapshot-runner-app/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+/*!
```

Comment:
I'm not good at naming, but I feel like we can come up with better name for this package.
Mabe ask question in Teams on suggestions?
I'd suggest putting better description in readme.md in this PR, and simply reference to it in such post, i.e. it should be sufficient to describe what it's about not to require more explanations in chat.

As for name, I have this tension - on one hand, implementation is ODSP specific, so maybe name of package should reflect that. On other hand, I can see a world where we will make it more generic (and maybe pluggable) in the future to support other storages. Then notion of "snapshot" should be removed, as it's rather specific to ODSP (we might use other representations, for example fluid-fetch representation that represents each blob as a file)

I'd try to find some generic names that do not put us in the corner in the future.
Maybe "fluid runner"? Does not tell much, but sort of in line with "fluid fetcher" and "replay tool" - you need to know more to tell what it actually does, but once you know, name resonates and sticks.